deploy/Dockerfile

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
Is this supposed to be a `Dockerfile` or the `entry.sh`? (check filename)

Reply:
Sorry, you're right, I've pushed the right files / names